signer.go

```diff
@@ -11,17 +10,16 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"crypto/x509/pkix"
-	"log"
 	"math/big"
 	"net"
 	"sort"
-	"strings"
 	"sync"
 	"time"
 )
 
 const signerVersion = ":inkfish1"
-const maxCertificateLifetimeDays = 84
+const certLifetimeMinutes = 240   // Set the NotAfter to be now + this value
```

Comment:
reason for such a short expiry? just so if problems it would be encountered more often and rolling it is cheap?

Reply:
OK, PR is quite a bit changed now, but yes, that's the rationale. Have run practical tests with cert regen every 4 minutes, seems to work OK.